geode-core/src/main/java/org/apache/geode/internal/cache/control/SerializableRegionRedundancyStatusImpl.java

```diff
@@ -53,7 +53,7 @@ public SerializableRegionRedundancyStatusImpl(PartitionedRegion region) {
    * @param region The region for which the lowest redundancy should be calculated.
    * @return The redundancy of the least redundant bucket in the region.
    */
-  private int calculateLowestRedundancy(PartitionedRegion region) {
+  int calculateLowestRedundancy(PartitionedRegion region) {
     int numBuckets = region.getPartitionAttributes().getTotalNumBuckets();
     int minRedundancy = Integer.MAX_VALUE;
```

Comment:
This seems complicated to me. Why not use 0? If I am wrong that is fine. I am happy to be wrong, but it doesn't seem like you need the MAX_VALUE construct. You are handling the -1 inside the loop.

Reply:
I was the original author of this, so I hope Alberto doesn't mind me answering for him. We're calculating the minimum redundancy level for a partitioned region, defined as the minimum redundancy level among all of its buckets, so given that any bucket may have a redundancy value between -1 (bucket does not exist) and N, the only sensible way to find the minimum is to start at the max value and reduce it as we find smaller values. If we started with the minimum redundancy at 0, we'd have no way of keeping track of what the lowest value for redundancy was among all the buckets.

Reply:
thanks for your answer Donal

Reply:
@mhansonp is it ok for you to remove the request for changes?

Reply:
@alb3rtobr I have approved your pull request.